README.md

```diff
@@ -127,6 +127,14 @@ qemu-system-x86_64 -nographic -kernel path/to/kernel -initrd /tmp/initramfs.linu
 Note that you do not have to build a special kernel on your own, it is
 sufficient to use an existing one. Usually you can find one in `/boot`.
 
+For framebuffer support, append a VESA mode via the `vga` kernel parameter:
```

Comment:
Perhaps add this to a different section further down titled "For framebuffer support in qemu".